src/Resources/contao/library/Contao/Database/Installer.php

```diff
@@ -331,26 +324,15 @@ public function getFromDca()
 	 */
 	public function getFromFile()
 	{
+		/** @var KernelInterface $kernel */
 		global $kernel;
 
 		$table = '';
 		$return = array();
 
 		// Only check the active modules (see #4541)
-		foreach ($kernel->getContainer()->get('contao.resource_provider')->getResourcesPaths() as $strModule => $strFolder)
+		foreach ($kernel->getContainer()->get('contao.resource_locator')->locate('config/database.sql') as $strFile)
 		{
-			if (in_array($strModule, array('calendar', 'comments', 'faq', 'listing', 'news', 'newsletter')))
```

Comment:
I still disagree. Did you think about that? The modules have been moved from `system/modules` to `vendor/contao/*-bundle`. Do you really think the `database.sql` will still be there?